cmd/operator/deploy/operator/crds.yaml

```diff
@@ -284,6 +284,9 @@ spec:
               type: object
               required:
                 - alerting
+                - labelLocation
```

Comment:
IIRC we wanted these to be specified at the OperatorConfig-level, so that one can specify this for both collectors and the rule-evaluator. I'll add a TODO.